### DISCUSSION TITLE
Add swipeable navigation

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#13291] [Ideas]
As a user, I would like to have swipeable navigation, similar to WeChat. This makes things so much easier, so I don't have to reach for a specific button in the top left corner to go back to the previous page.

Idea: On a chat page, You can swipe L>R to go back, and R>L to open the other person's profile, or if in a group open the group info page.

## Examples:

How WeChat does it:

https://user-images.githubusercontent.com/65834069/148661245-4aeac22f-bb54-477b-af72-8098a79d0365.mp4

How GETTR does it:

https://user-images.githubusercontent.com/65834069/148661429-0aa29906-5282-48c1-95ef-3eaa7bea8d4f.mp4

**Comment by hesterbruikman** [comment]
@ducheng0 thanks for raising this issue! There's a lot of opportunity for more gesture based / swipe interactions. We do need to make sure gestures don't conflict. E.g. swipe in a message in a 1:1 chat can easily be misinterpreted as swipe between nav bar items (swipe to wallet) and swipe to the other user's profile.

I think the issue would benefit from a global review of swipe actions and navigation. 

cc @pedro-et @xAlisher
